.eslint-dictionary.json

```diff
@@ -89,6 +90,7 @@
   "createcipheriv",
   "creds",
   "cron",
+  "customizations",
```

Comment:
This is not an English word, can `customization` be used instead?

Reply:
Updated